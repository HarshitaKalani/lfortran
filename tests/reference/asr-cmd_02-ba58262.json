{
    "basename": "asr-cmd_02-ba58262",
    "cmd": "lfortran --show-asr --no-color {infile} -o {outfile}",
    "infile": "tests/../integration_tests/cmd_02.f90",
    "infile_hash": "fc31fcafc39673c2cf4b192ea1cca649f864901bbc9b910f442cb5f9",
    "outfile": null,
    "outfile_hash": null,
    "stdout": "asr-cmd_02-ba58262.stdout",
    "stdout_hash": "4d10f2717c033b9435225173c283a4ae2497949e8eed9f832853cb8b",
    "stderr": null,
    "stderr_hash": null,
    "returncode": 0
}