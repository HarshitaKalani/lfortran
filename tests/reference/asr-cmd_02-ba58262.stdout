(TranslationUnit
    (SymbolTable
        1
        {
            lfortran_intrinsic_builtin:
                (IntrinsicModule lfortran_intrinsic_builtin),
            test_exec:
                (Program
                    (SymbolTable
                        2
                        {
                            execute_command_line:
                                (ExternalSymbol
                                    2
                                    execute_command_line
                                    4 execute_command_line
                                    lfortran_intrinsic_builtin
                                    []
                                    execute_command_line
                                    Private
                                ),
                            get_environment_variable:
                                (ExternalSymbol
                                    2
                                    get_environment_variable
                                    4 get_environment_variable
                                    lfortran_intrinsic_builtin
                                    []
                                    get_environment_variable
                                    Private
                                ),
                            homedir:
                                (Variable
                                    2
                                    homedir
                                    []
                                    Local
                                    ()
                                    ()
                                    Default
                                    (Character 1 255 ())
                                    ()
                                    Source
                                    Public
                                    Required
                                    .false.
                                )
                        })
                    test_exec
                    []
                    [(SubroutineCall
                        2 get_environment_variable
                        ()
                        [((StringConstant
                            "HOME"
                            (Character 1 4 ())
                        ))
                        ((Var 2 homedir))
                        (())
                        (())
                        (())]
                        ()
                    )
                    (Print
                        [(IntrinsicElementalFunction
                            StringTrim
                            [(Var 2 homedir)]
                            0
                            (Character 1 255 ())
                            ()
                        )]
                        ()
                        ()
                    )
                    (SubroutineCall
                        2 execute_command_line
                        ()
                        [((StringConstant
                            "printenv"
                            (Character 1 8 ())
                        ))
                        (())
                        (())
                        (())
                        (())]
                        ()
                    )]
                )
        })
    []
)
